src/util/psbt/map/input.rs

```diff
@@ -25,6 +25,34 @@ use util::psbt::map::Map;
 use util::psbt::raw;
 use util::psbt::serialize::Deserialize;
 use util::psbt::{Error, error};
+
+/// Type: Non-Witness UTXO PSBT_IN_NON_WITNESS_UTXO = 0x00
+const PSBT_IN_NON_WITNESS_UTXO: u8 = 0x00;
```

Comment:
This really looks like it wants to be an `enum`.

Reply:
With enum in rust we will have to add conversions from `u8` and into it as a manual impl; which will lead us to multiple use of the same constants in different pieces of the code, which is more error-prone when the new types are added

Reply:
Agreed, unfortunately Rust enums are not very good for "an open-ended list of `u8` constants"